common/lib/xmodule/xmodule/css/video/display.scss

```diff
@@ -55,7 +55,7 @@ div.video {
         float: left;
         border-radius: 3px;
         background-color: $very-light-text;
-        padding: ($baseline*0.75);
+        padding: ($baseline*0.65);
```

Comment:
This seems like it would not give a clean number. If you need specific padding, I'd just define it as a number here.